style.css

```diff
@@ -1 +1,66 @@
-/* No CSS *//*# sourceMappingURL=style.css.map */
\ No newline at end of file
+body {
```

Comment:
what about sass?

Reply:
sass is ignored by git

Reply:
it shouldn't ;) css is generated so it should be ignored